auslib/db.py

```diff
@@ -1156,7 +1176,8 @@ def insert(self, changed_by, transaction=None, dryrun=False, **columns):
             if not self._dataVersionsAreSynced(sc_id, transaction):
                 raise MismatchedDataVersionError("Conditions data version is out of sync with main table for sc_id %s",
                                                  sc_id)
-            self.auto_signoff(changed_by, transaction, sc_id, dryrun, columns)
+            if allow_auto_signoff:
```

Comment:
Can you explain why auto signoff shouldn't be allowed here? If I'm understanding correctly, users in one of the required groups could still sign off manually afterwards...